launch-game.sh

```diff
@@ -25,11 +25,21 @@ mono OpenRA.Game.exe Engine.LaunchPath="$MODLAUNCHER" $MODARG "$@"
 
 # Show a crash dialog if something went wrong
 if [ $? != 0 ] && [ $? != 1 ]; then
-	ERROR_MESSAGE="OpenRA has encountered a fatal error.\nPlease refer to the crash logs and FAQ for more information.\n\nLog files are located in ~/.openra/Logs\nThe FAQ is available at http://wiki.openra.net/FAQ"
+	# Match with X${OSTYPE} to silence a shell-check warning because ${OSTYPE} is not defined in posix
+	if expr match "X${OSTYPE}" "XDarwin.*" >/dev/null; then
```

Comment:
From IRC discussion: we could simplify this by instead checking for `if command -v osascript > /dev/null; then` (the macOS command to run applescript), and then actually using `osascript` to show a gui error dialog instead of falling back to the commandline.

Reply:
On second thought, lets not. Applescript is awful.